session_pool_test.go

```diff
@@ -0,0 +1,315 @@
+//go:build integration
+// +build integration
+
+/*
+ *
+ * Copyright (c) 2022 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License.
+ *
+ */
+package nebula_go
+
+import (
+	"fmt"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSessionPoolInvalidConfig(t *testing.T) {
+	hostAddress := HostAddress{Host: address, Port: port}
+
+	// No space name
+	_, err := NewSessionPoolConf(
```

Comment:
could use a slice

Reply:
The check is done during the construction, so we cant pre-build a slice of configs and check them one by one.